core/cap-0057.md

```diff
@@ -930,7 +1078,8 @@ persisted on validators via `lastArchivalEpochPersisted`. This will be useful to
 as it is the cutoff point at which a proof will need to be generated for `RestoreFootprintOp`.
 
 Whenever a `PERSISTENT` entry is evicted (i.e. removed from the Live State BucketList and added to the Hot Archive),
-the full entry will be emitted via `evictedPersistentLedgerEntries`.
+the entry key and its associated TTL key will be emitted via `evictedTemporaryLedgerKeys` (this field is unfortunately
+named for legacy reasons).
```

Comment:
I wonder if we can just rename the field; the rename won't change the XDR compatibility, will it? There also shouldn't be consumers of this field either, as we don't populate it currently.

Reply:
Good point, especially since we need a new `LedgerCloseMeta` version anyway.